appservice/src/SiteWrapper.ts

```diff
@@ -70,8 +70,13 @@ export class SiteWrapper {
     }
 
     public async deployZip(fsPath: string, client: WebSiteManagementClient, outputChannel: vscode.OutputChannel): Promise<void> {
+        const yes: string = 'Yes';
+        const warning: string = `Are you sure you want to deploy to '${this.appName}'? This will overwrite any previous deployment and cannot be undone.`;
```

Comment:
Should we add this prompt to App Services?

Reply:
App services calls this function, so no need to change that repo